.github/workflows/release.yaml

```diff
@@ -55,3 +55,14 @@ jobs:
 
       - name: Create deb repository
         run: ci/deploy-deb.sh
+
+      - id: get_version
+        uses: battila7/get-version-action@v2
```

Comment:
ditto.
https://stackoverflow.com/questions/71630227/github-action-how-to-remove-starting-v-from-variable